src/common/libflux/info.c

```diff
@@ -62,13 +62,12 @@ char *flux_getattr (flux_t h, int rank, const char *name)
     return ret;
 }
 
-int flux_info (flux_t h, int *rankp, int *sizep, bool *treerootp)
+int flux_info (flux_t h, uint32_t *rankp, uint32_t *sizep, int *arityp)
 {
     flux_rpc_t *r = NULL;
     JSON out = NULL;
     const char *json_str;
-    int rank, size;
-    bool treeroot;
+    int arity, rank, size;
```

Comment:
Just out of curiosity. Are we supporting both k-ary and k-nomial topologies or only k-ary at this point?